docs/articles_en/openvino-workflow/running-inference/model-input-output/dynamic-shapes.rst

```diff
@@ -40,12 +40,13 @@ The methods are sensitive to model internals, do not always give optimal perform
 For a short overview of the methods, refer to the :doc:`When Dynamic Shapes API is Not Applicable <dynamic-shapes/openvino-without-dynamic-shapes-api>` page.
 Apply those methods only if native dynamic shape API described in the following sections does not work or does not perform as expected.
 
-The decision about using dynamic shapes should be based on proper benchmarking of a real application with real data.
-Unlike statically shaped models, dynamically shaped ones require different inference time, depending on input data shape or input tensor content.
-Furthermore, using the dynamic shapes can bring more overheads in memory and running time of each inference call depending on hardware plugin and model used.
+It is recommended to benchmark your application with real data to determine if dynamic shapes are necessary and to assess their impact on performance and resource usage. It is because models with dynamic shapes may have different inference performance characteristics and memory requirements compared to statically shaped models. The overhead depends on the hardware plugin (that is, CPU, GPU or NPU) and the specific model.
 
-Handling Dynamic Shapes
-#######################
+.. note::
+    **GPU Dynamic Shape Support:** While GPU supports dynamic shapes, optimization is still in progress for a broader range of network architectures. Performance may vary depending on the specific model and use case. Consider testing with your specific workload to evaluate performance.
```

Comment:
```suggestion
.. note::

    **GPU Dynamic Shape Support:** GPUs support dynamic shapes, but optimization is still in progress for a broader range of models. Performance may vary depending on the specific model and use case. Consider testing with your specific workload to evaluate performance.
```

Reply:
small wording simplifications and a line after note directive 
very nicely done!!

Reply:
thanks for comment.